### ISSUE TITLE
Deleting datasets fails due to SQL syntax issue for PostgreSQL

### ISSUE THREAD

**Issue by kay73** [epam/NGB#737]
**Describe the bug**
Deleting datasets fails due to SQL syntax issue for PostgreSQL.

**Prerequisite**
- Dataset/project exists.
- Valid JWT token is used.
- ngb-cli has valid ngb server endpoint configuration and works otherwise.

**To Reproduce**

1. Try deleting a dataset, e. g.:
```
ngb dd 2
```
Fails reproducibly regardless if numeric ids or symbolic names are used. Tested for datasets that exist and appear when queried. 

2. Receive HTTP response `200 OK` with payload
```
{
    "message": "Your operation has been aborted due to incorrect grammar of a query.",
    "status": "ERROR"
}
```

**Expected behavior**
Indication that the project has been deleted successfully.

**Relevant stacktrace**
```
FJan 18 18:30:07 neodb-a10 ngb.sh[3889529]: 2022-01-18 18:30:07.393 ERROR 3889529 --- [nio-8080-exec-3] c.e.c.controller.ExceptionHandlerAdvice  : This operation has been aborted: uri=/catgenome/restapi/project/1;client=<REDACTED>;session=<REDACTED>;user=<REDACTED>.
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]: org.springframework.jdbc.BadSqlGrammarException: PreparedStatementCallback; bad SQL grammar [
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:                     DELETE catgenome.project_description
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:                     WHERE
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:                         project_id = ?
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:             ]; nested exception is org.postgresql.util.PSQLException: ERROR: syntax error at or near "catgenome"
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:   Position: 46
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:         at org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:231) ~[spring-jdbc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:         at org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73) ~[spring-jdbc-4.3.7.RELEASE.jar!/:4.3.7.RELEASE]
...
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]: Caused by: org.postgresql.util.PSQLException: ERROR: syntax error at or near "catgenome"
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:   Position: 46
Jan 18 18:30:07 neodb-a10 ngb.sh[3889529]:         at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182) ~[postgresql-9.4-1206-jdbc4.jar!/:9.4]
```
**Environment:**
 - NGB Version: 
   ```
   catgenome-psql-2.6.1.3260.jar
   2.6.1.3261.8e61fae2d3ecffea57598772e95aa2af7b77b815
   ```
 - PostgreSQL:
   ```
   [ngb]# yum info postgresql
   Name         : postgresql
   Version      : 10.15
   ```

**Comment by mzueva** [comment]
@kay73 Thank you for the issue! It shall be resolved via 7f564bb04b370aa8af71ed90e4441806ee56a253. Please find the development build with fix [3277](https://ngb-oss-builds.s3.amazonaws.com/web/index.html?prefix=public/builds/develop/2.6.1.3277/).

**Comment by mzueva** [comment]
Fixed
